### DISCUSSION TITLE
Kotlin server implementation

### DISCUSSION THREAD

**Discussion by smaye81** [connectrpc/.github#10] [General]
Would you or your company use a Connect **server** implementation in Kotlin? If yes, please leave a comment and tell us more about what you'd like: synchronous or asynchronous, important framework integrations (Spring, Vert.x, etc.), and anything else important.

**Comment by mcxinyu** [comment]
please oh please! :)

**Comment by rafaelfess** [comment]
I would use it with Ktor and ActiveJ. Ktor framework integration would be important!

**Comment by ichizero** [comment]
We would like to use a Connect server with Ktor. 
To enable a gradual migration, we expect the REST API and Connect Protocol to coexist within a single Ktor Server like connect-go's compatibility with `net/http`!

```mermaid
flowchart LR
    FE[Frontend React]
    
    subgraph Single Ktor Server
        REST[REST API]
        Connect[Connect]
    end
    
    FE -- "react-query" --> REST
    FE -- "connect-query-es" --> Connect
```

**Comment by franz-riptides** [comment]
A ktor connect server would be great!
